doc/index.rst

```diff
@@ -22,6 +22,13 @@ Zephyr Project Documentation
 
    `Zephyr 1.5.0`_ | `Zephyr 1.6.0`_ | `Zephyr 1.7.0`_ | `Zephyr 1.8.0`_
 
+For more information about previous releases, please consult the published
+:ref:`zephyr_release_notes`.
+
+The Zephyr OS imports or reuses packages, scripts and other files that
```

Comment:
Maybe start with:

     The Zephyr OS is provided under the :download:`Apache 2.0 license. <../LICENSE>`.
     However, the Zephyr OS imports...